### ISSUE TITLE
Disclosure buttons for new test queue tables missing aria-expanded="false" on page load

### ISSUE THREAD

**Issue by mcking65** [w3c/aria-at-app#1103]
The buttons that show/hide the test queue entry for a test plan version in the new test queue do not get announced as collapsed when the page loads.

To reproduce:
1. Load the [new test queue in the sandbox](https://aria-at-app-sandbox.bocoup.com/test-queue-2).
2. Focus a button under the heading for a test plan, e.g., under heading, "Action Menu Button Example Using aria-activedescendant", focus the button named "V22.03.17   Candidate".

Expected behavior: The button has aria-expanded=false so that it is announced by screen readers as collapsed.

Actual behavior: aria-expanded is not present until the button is activated. If activated to expand, aria-expanded is set to true. Then, if activated again, aria-expanded attribute is set to false.
